plugins/knife/knife-deployment-resize.rb

```diff
@@ -230,27 +230,16 @@ def run
       new_counts = Hash.new(0)
       new_counts["ellis-site1".to_sym] = 1
       new_counts[:seagull] = seagull_count || old_counts[:seagull]
-      %w{bono homer homestead sprout vellum dime}.each do |node|
+      %w{bono homer sprout vellum dime}.each do |node|
         for i in 1..number_of_sites
           new_counts["#{node}-site#{i}".to_sym] = count["#{node}-site#{i}".to_sym] || [old_counts["#{node}-site#{i}".to_sym], 1].max
         end
       end
-      %w{ibcf sipp ralf}.each do |node|
+      %w{ibcf sipp}.each do |node|
         for i in 1..number_of_sites
           new_counts["#{node}-site#{i}".to_sym] = count["#{node}-site#{i}".to_sym] || old_counts["#{node}-site#{i}".to_sym]
         end
       end
-      if attributes["split_storage"]
-        for i in 1..number_of_sites
-          new_counts.delete("homestead-site#{i}".to_sym)
```

Comment:
Because split_storage isn't an option anymore. So we could be keeping the first branch of the `if` block:

```
for i in 1..number_of_sites
  new_counts.delete("homestead-site#{i}".to_sym)
  new_counts.delete("ralf-site#{i}".to_sym)
end
```
But all this does is remove the homestead and ralf nodes from `new_counts`. But there's a change above which means we never add homestead or ralf to `new_counts`, so we don't need to remove them